moped-editor/src/views/projects/projectView/ProjectComponents/utils/form.js

```diff
@@ -386,7 +394,7 @@ export const useResetDependentFieldOnParentFieldChange = ({
       return;
     }
 
-    setValue(dependentFieldName, valueToSet);
+    setValue(dependentFieldName, valueToSet, { shouldValidate: true });
```

Comment:
This was a TIL for me. when we clear the completion date programmatically when the phase option is changed, the validation is triggered. This has the same effect as when any input in the form changes by a user (`mode: "onChange"` in the config for `useForm`).

See https://react-hook-form.com/docs/useform/setvalue